models/experimental/functional_stable_diffusion/demo/demo.py

```diff
@@ -53,10 +53,11 @@ def constant_prop_time_embeddings(timesteps, sample, time_proj):
     return t_emb
 
 
-def tt_guide(noise_pred, guidance_scale):  # will return latents
-    noise_pred_uncond, noise_pred_text = ttnn.split(noise_pred, noise_pred.shape[0] // 2, dim=0)
+def tt_guide(noise_pred_, guidance_scale):  # will return latents
+    noise_pred = ttnn.to_torch(noise_pred_)
```

Comment:
I have a solution in https://github.com/tenstorrent/tt-metal/pull/8843/ that should eliminate the need to go to torch.

Reply:
Removed to torch. Used your solution.